### ISSUE TITLE
data split

### ISSUE THREAD

**Issue by XLLXO** [hailanyi/VirConv#59]
Hello author!
I want to ask a question:
In order to submit to KITTI and obtain better test results
When dividing training set and validation set
If I use 80%-20% data split or 90%-10% data split
How to select training samples and validation samples?
Do you have a txt file that has been divided into training set and validation set? Can you share it?
This is my email:1102497242@qq.com
Looking forward to your reply
Thank you so much
